studio/lib/helpers.ts

```diff
@@ -165,33 +165,57 @@ export const copyToClipboard = (str: string, callback = () => {}) => {
   }
 }
 
-export async function passwordStrength(value: string) {
+type PasswordStrengthResponse = {
+  message: string
+  warning: string | null
+  strength: number
+}
+export async function passwordStrength(value: string): Promise<PasswordStrengthResponse> {
   let message = ''
   let warning = ''
   let strength = 0
 
-  if (value && value !== '') {
-    const response = await post(`${API_URL}/profile/password-check`, { password: value })
-    if (!response.error) {
-      const { result } = response
-      const score = (PASSWORD_STRENGTH as any)[result.score]
-      const suggestions = result.feedback?.suggestions ? result.feedback.suggestions.join(' ') : ''
-
-      // set message :string
-      message = `${score} ${suggestions}`
+  if (!value) {
+    return {
+      message,
+      warning,
+      strength,
+    }
+  }
 
-      // set strength :number
-      strength = result.score
+  if (value.length >= 100) {
+    return {
+      message: `${PASSWORD_STRENGTH[0]} Passwords must be less than 100 characters long.`,
```

Comment:
Updated this message to something that I think is clearer than the previous.  Previously I had `${PASSWORD_STRENGTH[0]} Passwords must be 99 or fewer characters.`